conf_publisher/confluence_api.py

```diff
@@ -26,7 +26,8 @@ def __init__(self, url, auth):
         self.auth = auth
         self.headers = {
             'content-type': 'application/json',
-            'Authorization': 'Basic ' + auth
+            'Authorization': 'Basic ' + auth,
+            'X-Atlassian-Token': 'no-check'
```

Comment:
Instead of passing `X-Atlassian-Token` header for each request consider
passing it directly in `self._api._get('{url}/pages/movepage.action...`.